src/python/library/tritonclient/http/_client.py

```diff
@@ -214,6 +217,13 @@ def _get(self, request_uri, headers, query_params):
         geventhttpclient.response.HTTPSocketPoolResponse
             The response from server.
         """
+        request = Request(headers)
+
+        # Call the triton client plugin
```

Comment:
should we remove the comments here to be consistent with grpc comments?

Reply:
Removed.